lib/metadata/network_interface.go

```diff
@@ -58,4 +58,7 @@ type ContainerNetwork struct {
 
 	// The IP ranges for this network
 	Pools []ip.Range `vic:"0.1" scope:"read-only" key:"pools"`
+
+	// set of network wide links and aliases for this container on this network
+	Aliases []string `vic:"0.1" scope:"hidden" key:"aliases"`
```

Comment:
In that case how do we differentiate network wide alias from container specific alias? It looks like it's
`target:alias` and therefore a global will be `:alias`?